h2/src/main/org/h2/mvstore/tx/VersionedValueCommitted.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2004-2017 H2 Group. Multiple-Licensed under the MPL 2.0,
```

Comment:
2004-2018

The same problem in `VersionedValueType`, `VersionedValueUncommitted`, and `VersionedValue`.